modules/social_features/social_landing_page/social_landing_page.install

```diff
@@ -82,8 +83,16 @@ function _social_landing_page_get_permissions($role) {
  * Enable the Social Featured Items module.
  */
 function social_landing_page_update_8801() {
-  $module = 'social_featured_items';
+  _social_landing_page_install_module('social_featured_items');
+}
 
+/**
+ * Install a separate module of the moved section.
+ *
+ * @param string $module
+ *   The module name.
+ */
+function _social_landing_page_install_module($module) {
```

Comment:
I don't think we should create a separate function here but should instead opt for code duplication.

A separate function indicates that this is something that can be done more often and encourages re-using the solution. Duplicating the code shows that it was a solution used for a very specific case (which this is).

Duplicating the code also makes cleaning up old update hooks easier in the future.

Reply:
Done